services/admin/assets/js/admin.js

```diff
@@ -343,7 +343,18 @@ var pretty = function(val) {
     }
 }
 
-var getVersion = function () {
+var getClientVersion = function () {
+    var query = $.get(window.location.origin + "/");
+
+    query.fail(handleRequestError);
+
+    query.done(function (data, status, xhr) {
+        var version = xhr.getResponseHeader('X-InfluxDB-Version');
+        $('.influxdb-client-version').html('v'+version);
```

Comment:
`X-InfluxDB-Version` returns `unknown` when InfluxDB is built without a version, which causes this to display as `InfluxDB vunkown`.